script.js

```diff
@@ -1,4 +1,218 @@
+// define options for game mode
+var gameStateArray = [
+  "RollDiceModePlayerOne",
+  "ChooseDiceOrderPlayerOne",
+  "RollDiceModePlayerTwo",
+  "ChooseDiceOrderPlayerTwo",
+];
+
+// set game mode to Player One's Dice Rolls
+var gameMode = gameStateArray[0];
```

Comment:
Nice, I like that you are defining your state variables and trying to group them in an array that train of thought is good so keep it up. 
Now to bring it to the next level, if you look at line 10 where you go `gameMode = gameStateArray[0]` yes, you have achieved the purpose of typing once and reducing typos with declaring the state variables above but if we just look at line 10, it is not immediately visible what the state you are setting it to because it is just `gameStateArray[0]`. What I am trying to say is that on top of organizing your code, try as much as possible to make each line understandable by itself!